frontend/scss/_variables.scss

```diff
@@ -1,17 +1,28 @@
-﻿@import "../node_modules/bootstrap/scss/variables";
```

Comment:
This file used to be two files.

Bootstrap says that all default variable overrides need to be done _before_ importing variables from Bootstrap.